packages/api/src/controllers/baseController.js

```diff
@@ -268,19 +268,4 @@ export default {
 
     return query.first();
   },
-
```

Comment:
Extracted from baseController to general `/utils` so it could be used in models as well (upon reflection it never needed to be in a controller to start with)